src/components/Loading.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {css as _css, keyframes} from 'styled-components';
+import css from '@styled-system/css';
+import {Box as BoxPrimitive, BoxProps as BoxPrimitiveProps} from 'mdlz-prmtz';
+
+type LoadingProps = BoxPrimitiveProps & {
+  progress?: number;
+};
+
+const loadingAnimation = (props: LoadingProps) => keyframes`
+  50% {
+    width: 100%;
+    margin-left: 0;
+  }
+  100% {
+    width: 0%;
+    margin-left: 100%;
+  }
+`;
+
+export const Loading = React.forwardRef<HTMLDivElement, LoadingProps>((props, ref) => (
+  <BoxPrimitive {...props} ref={ref} css={[
+    css({
+      height: '4px',
+      '::after': {
+        content: `""`,
```

Comment:
It seems like if we fill the content with just `''` instead of escaping it, the property is ignored by `css()` and the `::after` pseudo-element is not placed on the DOM (since it needs a `content` property to be defined), and because of that the loading bar disappears.